apps/wallet/src/ui/app/staking/home/ValidatorDataTypes.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//TODO pull from the SDK
```

Comment:
I think Jordan has added similar types to SDK, can we use those?

Reply:
I think so, but I am not sure if it has landed

Reply:
It hasn't landed, I'm trying to land a much larger schema change first, will probably land it later.